react-native-fbsdklogin/js/FBSDKLoginManager.ios.js

```diff
@@ -51,63 +57,52 @@ type FBSDKDefaultAudience = $Enum<{
   'only-me': string,
 }>;
 
-/**
- * Provides methods for logging the user in and out.
- */
-class FBSDKLoginManager {
+module.exports = {
```

Comment:
this seems to just wrap native calls. Should we instead just provide flow interface definitions?